docs/contribution/release-schedule.md

```diff
@@ -0,0 +1,104 @@
+
+> Dates are subject to change
+
+![schedule][release-schedule-static]
+
+```
+      Jul 2023   Jun 2024                Oct 2024             Nov 2024             Dec 2024
+     +---------------------------------------------------------------------------------------------+
+main | ## Unstable ############################################################################### |
+     +---------------------------------------------------------------------------------------------+
+v1   | ## Active ## / ## Maintenance ########################### /                                 |
+     +---------------------------------------------------------------------------------------------+
+v2   |              / ## Active ################################ / ## Maintenance ################ |
+     +---------------------------------------------------------------------------------------------+
+v3   |                                     / ## Prerelease ##### / ## Active ##################### |
+     +---------------------------------------------------------------------------------------------+
+v4   |                                                                                             |
+     +---------------------------------------------------------------------------------------------+
+```
+
+## Next Major Release Plans
+
+👉 See [Releases in Jira Board][release-jira-board].
+
+## Release Phases
+
+### Prerelease
+
+The prerelease phase is intended to be the opportunity for early adopters,
+library authors, and other strategic ecosystem partners to begin to evaluate and
+integrate new changes into their codebases. For v3, this phase was more than a one month
+long and spanned five prerelease/alpha releases. We hope to extend this timeframe
+even further for our next major.
+
+### Active
+
+Consuming projects should always aim to follow the Active release.
+
+A release in the Active phase receives biweekly minor releases containing new
+features and fixes. The work we deliver into `main` every day is considered
+unstable. Almost every two weeks we package up these changes into a new minor version
+that is published from `main` to the current Active major.
+
+### Maintenance
+
+For a release in the Maintenance phase, patch releases are published containing
+security patches and critical bug fixes. When a version moves from Active to
+Maintenance, consuming projects should begin migrating to the new Active major
+version. During Maintenance we also consider adding non-critical bug fixes on an
+ad hoc basis, by request only. To request a fix be back-ported to a Maintenance
+release, please
+[open an issue][spirit-jira-board] or let us know in [support Slack channel][spirit-support-slack-channel].
+
+Changes required for critical security and bug fixes may lead to semver-major
+changes landing within a release stream, such situations will be rare and will
+land as semver-minor. Although, those changes should have a revert option
```

Comment:
> will land as semver-minor

I understand (or I hope I do), but do we want to do that…? Does it mean a security fix may require changes in our users' code base?

Reply:
I think if that would be so critical and crucial it should be that way. We have a process for BC that requires a lot of work and documents to be created. So we have to have an exception fot this process. And shit sometimes happen 🤷‍♂️